templates/api-server.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "api-server",  
+  "description": "A LoopBack API server with local User auth",
+  "package": {
+    "version": "1.0.0",
+    "main": "server/server.js",
+    "scripts": {
+      "lint": "eslint .",
+      "start": "node .",
```

Comment:
@bajtos @superkhau do you like the idea of having the template as a json file. the original workspace had it as a folder with .js files